docs/howto/object-storage/s3/versioning.md

```diff
@@ -44,6 +44,61 @@ the following commands:
     This functionality is not available with the `s3cmd` command.
 
 
+## Creating a versioned object
+
+Once object versioning is enabled on a bucket, the normal object
+creation and replacement commands behave in a manner different from
+that in unversioned buckets:
+
+* If the object does not already exist, it is created (as in an
+  unversioned bucket).
+* If the object does already exist, it is not replaced. Instead, a new
+  version appears, in addition to the old one.
+
+=== "aws"
+    ```bash
+    aws --profile <region> \
+      --endpoint-url=https://s3-<region>.{{extra.brand_domain}}:8080 \
+      s3api put-object \
+      --bucket <bucket-name> \
+      --key <object-name> \
+      --body <local-filename>
+    ```
+=== "mc"
+    ```bash
+    mc cp \
+      <local-filename> \
+      <region>/<bucket-name>/<object-name>
+    ```
+=== "s3cmd"
+    ```bash
+    s3cmd put <local-filename> s3://<bucket>
+    ```
+
+
+## Listing object versions
+
+If a bucket that has versioning enabled, you may list the versions
```

Comment:
Minor typo, this sentence may start like "In a bucket that has versioning" or like "If a bucket has versioning".

Reply:
I meant "in". Thanks for spotting. :)